ownership-chain/runtime/src/tests.rs

```diff
@@ -61,3 +72,27 @@ fn asset_id_to_address_two_assets_same_owner() {
 		AccountId::from_str("c0f0f4ab324c46e55d02d0033343b4be8a55532d").unwrap()
 	);
 }
+
+#[test]
+fn minimum_balance_should_be_1() {
+	assert_eq!(Balances::minimum_balance(), 0);
+}
+
+#[test]
+fn deposit_minimum_amount_should_succeed() {
```

Comment:
```suggestion
fn deposit_smallest_unit_should_succeed() {
```

Reply:
I don't see any benefit here

Reply:
I just think `minimum_amount` is a bit vague. is it the minimum amount that you can deposit? Then depositing 0 will also work:

```
Balances::deposit(&alice, 0, Precision::Exact);
```

so, `smallest unit` makes more sense because it eliminates the possibility of deposit amount being zero